README.md

```diff
@@ -3956,8 +3938,6 @@ Bookmarks
 
 [Malware Unicorn 101](https://securedorg.github.io/RE101/)
```

Comment:
This should also be replaced, and with https://malwareunicorn.org/workshops/re101.html#0